identity-backend/src/user/repository/customer.model.ts

```diff
@@ -0,0 +1,21 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import mongoose, { HydratedDocument } from 'mongoose';
+import { Auth } from '../../auth/repository/auth.model';
+
+export type UserDocument = HydratedDocument<User>;
+
```

Comment:
Why you named the file as a cutomer, but Model named as User?

Reply:
my bad. Already fixed